### ISSUE TITLE
关于moving_avg超参数的疑问

### ISSUE THREAD

**Issue by 1348598339** [thuml/Autoformer#176]
请问一下，为什么按源代码的参乎上设置，当把moving_avg的值由25改为24，就会出现错误，moving_avg的取值要遵循什么样的原则，超参数调优的话，需要怎么设置moving_avg的范围

**Comment by 1348598339** [comment]
Run 43o69nwb errored: RuntimeError('The size of tensor a (168) must match the size of tensor b (192) at non-singleton dimension 1')，以上为调优过程中出现的问题，上述超参数为'activation': 'gelu', 'batch_size': 72, 'bucket_size': 4, 'c_out': 324, 'checkpoint': './checkpoint/', 'd_ff': 1024, 'd_layers': 5, 'd_model': 512, 'data': 'TEC_Encoded_2h', 'data_path': 'tec_features_encoded_2h_nan_normalized.csv', 'dec_in': 324, 'des': 'test', 'devices': '0,1,2,3', 'distil': False, 'do_predict': False, 'dropout': 0.24901082591657983, 'e_layers': 6, 'embed': 'timeF', 'enc_in': 324, 'factor': 2, 'features': 'M', 'freq': 'h', 'gpu': 0, 'is_training': 1, 'itr': 1, 'label_len': 168, 'learning_rate': 4.09977069459757e-05, 'loss': 'mse', 'lradj': 'type2', 'model': 'Autoformer', 'model_id': 'test', 'moving_avg': 25, 'n_hashes': 4, 'n_heads': 4, 'num_workers': 10, 'output_attention': 'store_true', 'patience': 5, 'project_name': 'autoformer_sweep', 'root_path': './data/TEC_Dataset/', 'seq_len': 144, 'target': 'F324', 'train_epochs': 1000, 'use_amp': False, 'use_gpu': True, 'use_multi_gpu': False}，请问这样的问题源头在哪里

**Comment by hugski** [comment]
应该是需要奇数, kernel_size必须为奇数.moving avg大小应该看你对数据平滑程度对要求了
